x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -1458,6 +1458,37 @@ m:i | o:i   | l:i | s:i
 1   | 39729 | 1   | 39729
 ;
 
+byTwoCalculatedSecondOverwrites
```

Comment:
nit: could we place these next to the `shadowing...` tests and use the `shadowing...` prefix to mark these tests? (Similarly in `inlinestats.csv-spec`) By now, each command has a suite of shadowing tests, and it'd be great to be able to easily find them later. (Maybe they deserve separate files?)